lib/tasks/team_project.rake

```diff
@@ -0,0 +1,14 @@
+namespace :team_project do
```

Comment:
Should we move this to `single_run.rake`? I would think we'd only run it once and can forget about it after that.